PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/common/classifiedexception/SparkToolException.kt

```diff
@@ -43,7 +44,12 @@ class SparkToolException(exp: Throwable?) : ClassifiedException(exp) {
 object SparkToolExceptionFactory : ClassifiedExceptionFactory() {
     override fun createClassifiedException(exp: Throwable?): ClassifiedException? {
         val stackTrace = if (exp != null) ExceptionUtils.getStackTrace(exp) else EmptyLog
-        return if (exp !is YarnDiagnosticsException && stackTrace.contains(ToolPackageSuffix)) {
+        return if (exp !is YarnDiagnosticsException
+                // Thrown from Azure blob storage SDK, refer to Issue #2580
+                && !(exp is IllegalArgumentException && stackTrace.contains("com.microsoft.azure.storage.CloudStorageAccount"))
+                // Thrown from creating Livy helper session to upload artifacts,refer to Issue #2552
```

Comment:
You can leave this to next fix as a new year gift. 🎁 

```suggestion
                // Thrown from creating Livy helper session to upload artifacts, refer to Issue #2552
```